### PR TITLE
When we submitt email address was displaying undefined, we forgot to …

### PR DESCRIPTION
**Pull request by Raagu1993**
…update variables in alert window. Now it is working fine. Please review and approve

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
                      var FirstName = document.getElementById('FirstName');
```
Comment (RakeshMaddala) [inline form.html]: we should follow variable naming conventions should be camel case.
```firstName``` instead of ```FirstName```

File: form.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
                     var LastName = document.getElementById('LastName');
```
Comment (RakeshMaddala) [inline form.html]: Follow naming convention, here as well

Comment (RakeshMaddala) [inline form.html]: @Raagu1993 - I have left two comments. Change those and push the code. I will approve then :)

Comment (Raagu1993) [reply form.html]: updated

Comment (RakeshMaddala) [reply form.html]: I think u have not pushed ur changes.
If u observe, we are having only one commit on this PR.

**Comment by Raagu1993** [comment]
[image: Inline image 1]

when I checked in class/update7, i can see changes. Could you please check
once

**Pull request closed without merging**